cmd/root.go

```diff
@@ -49,73 +60,49 @@ triage.`,
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
 	if err := rootCmd.Execute(); err != nil {
-		fmt.Println(err)
+		log.Println(err)
 		os.Exit(1)
 	}
 }
 
-// This function is useful for understanding what a particular viper contains.
-// It is more a crutch for development than anything I would ever expect a customer to see.
-func viperWiper(v *viper.Viper) {
-	fmt.Print("\n === Viper Wiper === \n\n")
-	for _, name := range v.AllKeys() {
-		fmt.Println("Key: ", name, " => Name:", v.GetString(name))
+func initConfig() {
+	if cfgFile != "" {
+		// log.Println("Using config file from the flag")
+	} else {
+		// log.Println("Using default config location")
+		cfgFile = "system_config.yaml"
 	}
```

Comment:
```
if len(cfgFile) == 0 {
    cfgFile = "system_config.yaml"
}
```

Reply:
That would work.  I figured it would be best to keep it to match the cobra docs so others can grok it better.

https://github.com/spf13/cobra/blob/8eaca5f0f49ad747a0722d39dca7a75c34abd21a/user_guide.md#create-rootcmd

I can change it if you really want though.

Reply:
Fwiw matching cobra docs wouldn't hurt. But the code Heemstra points out seems like an honest optimization 🤷🏽

Reply:
Yeah, I kept it like the cobra docs for now.  I think what I came up with works.  I do less checking if the file exists since we don't care as much.  I think this last push will address the other "invert logic" functions.